src/main/scala/viper/silver/ast/Ast.scala

```diff
@@ -397,6 +397,14 @@ case object Synthesized extends Info {
   lazy val isCached = false
 }
 
+/** An `Info` instance for labelling an AST node which is expected to fail verification.
+ * This is used by Silicon to avoid stopping verification.
+*/
+class ExpectFail extends Info {
+  lazy val comment = Nil
```

Comment:
My understanding is that, in order to avoid initialisation races, `lazy val`s ultimately use a lock behind the scene. Using `lazy val`s for cheap data (neither potentially space nor time consuming) is thus not recommended.

Reply:
Hmm strange that all of the other `Info`s also have `lazy val {comment,isCached}` - I'll go through and change those too. And maybe a full separate PR to remove other `lazy val`s from silver.